provisioning/app-proxy/kubernetes/templates/proxy/deployment.yaml

```diff
@@ -0,0 +1,125 @@
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: app-proxy
+  namespace: $NAMESPACE
+  labels:
+    app: app-proxy
+    releaseId: $RELEASE_ID
+spec:
+  replicas: $APP_PROXY_REPLICAS
+  selector:
+    matchLabels:
+      app: app-proxy
+  template:
+    metadata:
+      labels:
+        app: app-proxy
+        releaseId: $RELEASE_ID
+        tags.datadoghq.com/env: "$KEBOOLA_STACK"
+        tags.datadoghq.com/service: "app-proxy"
+        tags.datadoghq.com/version: "$RELEASE_ID"
+      annotations:
+        log: "true"
+        ad.datadoghq.com/tags: '{"git.commit.sha": "$COMMIT_HASH", "git.repository_url": "github.com/keboola/keboola-as-code"}'
+        ad.datadoghq.com/app-proxy.checks: |
+          {
+            "openmetrics": {
+              "instances": [
+                {
+                  "openmetrics_endpoint": "http://%%host%%:9000/metrics",
+                  "metrics": [".*"],
+                  "collect_histogram_buckets": true,
+                  "histogram_buckets_as_distributions": true,
+                  "tag_by_endpoint": false
+                }
+              ]
+            }
+          }
+    spec:
+      containers:
+        - name: app-proxy
+          image: $APP_PROXY_REPOSITORY:$APP_PROXY_IMAGE_TAG
+          resources:
+            requests:
+              cpu: "200m"
+              memory: "768Mi"
+            limits:
+              memory: "768Mi"
+          ports:
+            - containerPort: 8000 # Proxy
+            - containerPort: 9000 # metrics
+          env:
+            # 80% of the resources.limits.memory
+            - name: GOMEMLIMIT
+              value: "550MiB"
+            - name: APP_PROXY_SANDBOXES_API_HOST
+              valueFrom:
+                configMapKeyRef:
+                  name: app-proxy
+                  key: sandboxesApiHost
+            - name: APP_PROXY_PUBLIC_ADDRESS
+              valueFrom:
+                configMapKeyRef:
+                  name: app-proxy
+                  key: appProxyApiHost
+            - name: APP_PROXY_COOKIE_SECRET_SALT
+              valueFrom:
+                secretKeyRef:
+                  name: app-proxy
+                  key: cookieSecretSalt
+            - name: DD_AGENT_HOST
+              valueFrom:
+                fieldRef:
+                  fieldPath: status.hostIP
+            - name: DD_TRACE_ANALYTICS_ENABLED
+              value: "true"
+            - name: DD_PROFILING_ENABLED
+              value: "true"
```

Comment:
tato env je tu imo zbytocna, podla docs to ani nezapina ten profiling. Potrebujete vlastne ten DD profiling (plati sa zato zvlast)

Reply:
ok :) to je asi 5 generacia, kde sa to iba kopiruje, neviem, ktora bola povodna service

Reply:
Jachym, ked to budes vyhadzovat, tak to prosim vyhod z celeho repa.